packages/e2e-tests/test-applications/create-react-app/package.json

```diff
@@ -22,7 +22,11 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "test:build": "pnpm install && pnpm build",
+    "test:build-ts3.8": "pnpm install && pnpm add typescript@3.8 && pnpm build",
+    "test:build-canary": "pnpm install && pnpm add react@canary react-dom@canary && pnpm build",
+    "test:assert": "pnpm -v"
```

Comment:
I'm assuming we don't have actual tests in this app and this is just a placeholder command? (no action required, this is fine, just asking)

Reply:
yes, I figured it's easier to ensure we have _a_ command everywhere and leave this to the test app to just do nothing - `pnpm -v` was the easiest thing I could come up with 😅